rest/model/distro.go

```diff
@@ -8,8 +8,10 @@ import (
 // APIDistro is the model to be returned by the API whenever distros are fetched.
 // EVG-1717 will implement the remainder of the distro model.
 type APIDistro struct {
-	Name             APIString `json:"name"`
-	UserSpawnAllowed bool      `json:"user_spawn_allowed"`
+	Name             APIString               `json:"name"`
+	UserSpawnAllowed bool                    `json:"user_spawn_allowed"`
+	Provider         string                  `json:"provider"`
+	ProviderSettings *map[string]interface{} `json:"provider_settings"`
```

Comment:
I would make this not a pointer and dereference the field below

Reply:
Done